pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiActivityInstance.java

```diff
@@ -201,6 +201,13 @@ int updateOndemandTriggerId(@Bind("userId") long userId, @Bind("activityInstance
             + " order by ai.created_at desc limit 1")
     Optional<String> findLatestInstanceGuidByUserIdAndActivityId(@Bind("userId") long userId, @Bind("activityId") long activityId);
 
+    @SqlQuery("select ai.activity_instance_id from activity_instance as ai"
+            + "  join user as u on u.user_id = ai.participant_id"
+            + " where ai.study_activity_id = :activityId"
+            + "   and u.guid = :userGuid"
```

Comment:
should we check and eliminate hidden instances !!
like " and ai.is_hidden = false"

Reply:
I think `is_hidden` is mostly for UI display and such. The data is still available and accessible (and exported!).